app/flatpak-cli-transaction.c

```diff
@@ -870,7 +870,7 @@ transaction_ready (FlatpakTransaction *transaction)
   if (self->installing + self->updating + self->uninstalling > 1)
     {
       flatpak_table_printer_set_column_expand (printer, i, TRUE);
-      flatpak_table_printer_set_column_title (printer, i++, _("Change"));
+      flatpak_table_printer_set_column_title (printer, i++, _("Op"));
```

Comment:
We should probably add a translator comment here ? Say:

Translators, this is a shorthand for "Operation"